about/faq/index.html

```diff
@@ -152,7 +151,13 @@ <h3 id="q-how-to-clean-a-dirty-core-folder">Q: How to clean a dirty core folder?
 <h3 id="q-can-i-donate">Q: Can I donate?</h3>
 
 <p>We&rsquo;ve closed donations for now. If you have money burning a hole in your pocket though, consider donating to NotFood. She&rsquo;s
-our headcoder/maintainer. <a href="https://ko-fi.com/notfood">https://ko-fi.com/notfood</a></p>
+our headcoder/maintainer. <a href="https://ko-fi.com/notfood">https://ko-fi.com/notfood</a> or any of the numourous people who've helped out in the past: <br>
```

Comment:
Worded weirdly? Numerous is the correct spelling. Also i'd recommend rewriting this whole section, doesn't seem well-written. (Partly because of my old sentence structure.)